meta/main.yml

```diff
@@ -12,5 +12,3 @@ galaxy_info:
   categories:
     - cloud
     - system
-dependencies:
-  - kostyrevaa.foreman
```

Comment:
Sorry, this will not do.
I'm using kostyrevaa.foreman to setup yum repo.
On servers acting just as smart proxies I set [foreman_setup_only_repo](https://github.com/kostyrevaa/ansible-role-foreman/blob/master/defaults/main.yml#L3) to true